examples/quickstart/README.md

```diff
@@ -8,10 +8,10 @@ Then open [notebooks/quickstart.ipynb](notebooks/quickstart.ipynb) to get starte
 ## :earth_americas: Overview
 
 This quickstart aims to give you a small illustration of what ZenML can do. To do so, we will:
-- Train a model on the MNIST digits dataset, evaluate it, deploy it, and embed it in an inference pipeline
-- Automatically version, track, and cache data, models, and other artifacts
-- Track model hyperparameters and metrics in an experiment tracking tool
-- Measure and visualize train-test skew, training-serving skew, and data drift
+- Train a model on the iris flower classification dataset, evaluate it, deploy it, and embed it in an inference pipeline,
+- Automatically version, track, and cache data, models, and other artifacts,
+- Track model hyperparameters and metrics in an experiment tracking tool,
+- Measure and visualize train-test skew, training-serving skew, and data drift.
```

Comment:
Isn't this the grammatically correct way to format lists though?

Reply:
Technically, maybe, but only if you consider lists to be part of a whole long prose text.

I think it's also really common just to see lists without commas in between (i.e. because the bullet and the indent is doing the work of showing that 'this is a separate element / thing'.

It's a small point, though :) Do what makes you happiest ::)